zone/perl_client.cpp

```diff
@@ -3393,6 +3408,9 @@ void perl_register_client()
 	package.add("CashReward", &Perl_Client_CashReward);
 	package.add("ChangeLastName", &Perl_Client_ChangeLastName);
 	package.add("GrantPetNameChange", &Perl_Client_GrantPetNameChange);
+	package.add("GrantNameChange", (void(*)(Client*))&Perl_Client_GrantNameChange);
+	package.add("IsNameChangeAllowed", (bool(*)(Client*))&Perl_Client_IsNameChangeAllowed);
```

Comment:
We are trying to keep this list alpha ordered, I'd move IsNameChangeAllowed down